### DISCUSSION TITLE
How to install / config / use Fomantic UI in Gatsby?

### DISCUSSION THREAD

**Discussion by piscespieces** [fomantic/Fomantic-UI#1825] [General]
Hello there,

I got here thanks to reddit users talking about Semantic UI being dead when posted a question about how to use Semantic UI in a Gatsby project, since I haven't find an easy way to do it. There are some Medium posts and blog posts in general that walks you through a way to make it work, but it seemed to complicated when usually these UI libraries are pretty easy to install and use.

I would appreciate any help... Thanks!

**Comment by ko2in** [comment]
> Hello there,
> 
> I got here thanks to reddit users talking about Semantic UI being dead when posted a question about how to use Semantic UI in a Gatsby project, since I haven't find an easy way to do it. There are some Medium posts and blog posts in general that walks you through a way to make it work, but it seemed to complicated when usually these UI libraries are pretty easy to install and use.
> 
> I would appreciate any help... Thanks!

I'm not Gatsby user, but someone who knows Gatsby in FUI community would be helpful to you if you post what kind of issue you're facing.
